src/js/components/sidenav.js

```diff
@@ -89,31 +89,43 @@ export default class Sidenav {
   }
   
   init() {
+    // Add click handler
+    this.element.addEventListener('click', this._handleClick, false);
+
+    // Expand all if openAllOnInit option is set
+    if (this.openAllOnInit) {
```

Comment:
You'll need to move the `const menuToggles` and `const childMenus` declarations below up above this `if` check. Right now if someone passes in the `openAllOnInit` option as `true` those declarations will be unreachable, so when you do `menuToggles.forEach` it will error out.